build.py

```diff
@@ -522,7 +522,9 @@ def _likely_binary(sdist: str) -> str | None:
 
     ret = set()
     for name in names:
-        if "/test/" in name or "/tests/" in name:
+        if "/test/" in name \
+        or "/tests/" in name \
+        or "/examples/" in name:
```

Comment:
ideally python should never have backslashes -- always use parens for this

but actually we should just upgrade pycparser instead rather than allowing all `examples` directories -- I upgraded pycparser in sentry to avoid the same thing

Reply:
unfortunately salt-ssh 3004.2 has `- pycparser [required: ==2.19, installed: 2.19]`...

Reply:
salt-ssh 3005 has `- pycparser [required: ==2.21, installed: 2.21]` but not sure if we want 3005 yet